MultipleFileUpload/Model/MySQL/Queue.php

```diff
@@ -0,0 +1,162 @@
+<?php
```

Comment:
Nette\Database implementation was not in last release use Nette\Environments, I am reverting to my orignal solution -> when it will be approved, i will refactor all Models but firs i need to confirm MySQL namespace (later it can be removed)